packages/NodeTypeResolver/PHPStan/Scope/PHPStanNodeScopeResolver.php

```diff
@@ -166,10 +166,10 @@ public function processNodes(
             } elseif ($node instanceof PropertyFetch) {
                 $node->var->setAttribute(AttributeKey::SCOPE, $mutatingScope);
                 $node->name->setAttribute(AttributeKey::SCOPE, $mutatingScope);
-            } elseif ($node instanceof ConstFetch) {
-                $node->name->setAttribute(AttributeKey::SCOPE, $mutatingScope);
             } elseif ($node instanceof CallLike) {
                 $this->processCallike($node, $mutatingScope);
+            } elseif ($node instanceof ConstFetch) {
```

Comment:
I wonder why this changed. if not necessary I would drop it to reduce noise in the PR